client/app/components/notification/notification.controller.js

```diff
@@ -0,0 +1,36 @@
+class NotificationController {
+  constructor(User, Authentication, $state, $injector) {
+    "ngInject";
+    Object.assign(this, {
+      User,
+      Authentication,
+      $state,
+      $injector
+    });
+  }
+
+  isVerified() {
+    return this.userdata.mail_verified;
+  }
+  sendVerification(email) {
+    const $mdToast = this.$injector.get("$mdToast");
```

Comment:
Is there a reason why you inject $mdToast here? I would just inject it in the constructor (analog to `$state` etc)